src/app/pre-sale/components/popup/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Modal, ModalBody, ModalContent, ModalOverlay } from '@chakra-ui/react'
+
+import { PreSaleWidget } from '../pre-sale-widget'
+
+type PreSalePopupProps = {
+  isOpen: boolean
+  onClose: () => void
+}
+
+export const PreSalePopup = (props: PreSalePopupProps) => {
+  const { isOpen, onClose } = props
+  return (
+    <Modal onClose={onClose} isOpen={isOpen} isCentered scrollBehavior='inside'>
+      <ModalOverlay className='bg-ic-black bg-opacity-60 backdrop-blur' />
+      <ModalContent className='bg-transparent shadow-none'>
+        <ModalBody className='bg-transparent'>
+          <div className='flex flex-row'>
+            <PreSaleWidget />
+          </div>
+        </ModalBody>
+      </ModalContent>
+    </Modal>
+  )
+}
```

Comment:
Not sure if you saw my recent comment for onramping. Still using chakra component here as it's the quickest way but already added a ticket to the backlog where we could tackle to convert all modals/popups to a common style and component (tailwind).

Reply:
nvm saw you already commented. discussion offline.